311-status-protocol.md

```diff
@@ -0,0 +1,262 @@
+  transported.
+
+- III. Security - use state-of-the-art technologies, and research new security
+  methods and technologies to make strong security guarantees.
+
+- IV. Privacy - protect privacy in communication and transactions, and strive to
+  provide right to total anonymity.
+
+- V. Transparency - be open about what we are doing, allow community
+  contributions and be clear about trade-offs.
+
+- VI. Openness - specification is open and under a permissive license.
+
+- VII. Decentralization - p2p network, maximizing number of computers/humans who
+  can control and use the protocol we are building.
+
+- VIII. Inclusivity - easy to use our protocols, as well as working towards
+  interopability with other protocols, and educating users.
+
+- IX. Continuance - network should be incentivized to continue on its own.
+
+- X. Resourcefulness - work with other teams to avoid duplication of effort,
+  study prior work deeply to minimize wasted and duplicate effort.
+
+### Anonymous Communication Protocol requirements
+
+The purpose of an anonymous communication (AC) protocol is to ensure metadata
+isn’t leaked when messages are communicated between peers. It deals with how
+messages are transported, and not what is in them.
+
+This is the inital focus and work has begun together with Web3 Foundation and
+Validity Labs. Note that this list of requirements is tentative.
+
+1. Sender Anonymity
+2. Receiver Anonymity
+3. Sender-Receiver Unlinkability
+4. Reasonable Latency
+5. Reasonable Bandwidth
+6. Adaptable Anonymity
+7. Scalable
+8. No specialized Services
+
+Up for debate: incentives, network spam, asynchronous messaging.
+
+For more details and the latest updates,
+see [w3f/messaging](https://github.com/w3f/messaging).
+
+## Budget
+
+Headcount based. Small group.
+
+## Team / contributors
+
+### Status
+
+- Oskar
+- <1-2 protocol engineerings>
+
+Review/part-time, pending other commitments:
+- Jacek
+- Andrea MP
+- Corey
+- Jarrad
+
+### Anonymous Communication Protocol
+
+- Web3 Foundation
+- Status
+- Validity Labs
+- who else?
+
+## Scope
+The above plan is ambitious and the team size is currently limited. It is thus
+likely the scope will be limited, at least initially, to ensure solid progress
+in the most fruitful direction and avoid scope creep.
+
+Expect the precise scope and requirements to change soemwhat as we get more
+familiar with the problem domain, and depending on how research and
+collaborations with other teams go.
+
+## Milestones
+
+Hard to provide with research, but rough timeline to indicate effort.
+
+### Team formation, initial research, and rough project timeline
+
+#### November 2018 - December 2018
+
+#### Goals:
+1. Study state of the art deeply
+– includes p2p and networking basic literature
+– as well as anonymity/secure p2p specific (Tor, Briar, etc)
+2. Sketch out rough layers of concerns
+3. Team formation and collaboration with others in the space
+4. Do write-up of lessons learned so far
```

Comment:
I would love a presentation about lessons learned and the landscape on the Town Hall or a separate meeting, sounds super interesting to me (though I'm not ready to dig through all the papers myself atm).